components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1067,7 +1237,9 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
          vap_ice_exchange(its:ite,kts:kte),& ! OUT sum of vap-ice phase change tendencies
          qv_prev(its:ite,kts:kte),         & ! IN  qv at end of prev p3_main call   kg kg-1
          t_prev(its:ite,kts:kte),          & ! IN  t at end of prev p3_main call    K
-         col_location(its:ite,:3)          & ! IN column locations
+         col_location(its:ite,:3),          & ! IN column locations
+         do_prescribed_CCN,                 & !IN  .true. = prescribe CCN
+         nccn_prescribed(its:ite,kts:kte)    & !IN prescribed CCN concentration
          )
 
```

Comment:
In your original commitment, there is no "elapsed_s" argument also, I doubt that it is another issue caused your problem.